api/src/test/resources/io/strimzi/api/kafka/model/041-Crd-kafkaconnect.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why are these changing from `v1` to `v1beta1`?

Reply:
These are the resources used for tests. In the past, we had `apiextensions/v1beta1` in install and `apiextensions/v1` here. This PR turns it around => keeps `apiextensions/v1` in install and `apiextensions/v1beta1` here to have the old tests (for exampel the `api-conversion` tool tests etc.) still working. I think it is useful to keep them around and run the tests at least until the 0.23 release is out.